app/services/search/query_builders/chat_channel_membership.rb

```diff
@@ -6,6 +6,7 @@ class ChatChannelMembership < QueryBase
         channel_type
         status
         viewable_by
+        channel_discoverable
```

Comment:
Since we are not going to be using this field in Elasticsearch we should remove all the Elasticsearch code that was put in place for it.